setup.cfg

```
@@ -63,6 +63,7 @@ devpi_client =
 keyring.backends =
 	Windows = keyring.backends.Windows
 	macOS = keyring.backends.macOS
+	LibSecret = keyring.backends.libsecret
```

I don’t like the capitalization. The official name of libsecret is lowercase: https://wiki.gnome.org/Projects/Libsecret.

```suggestion
	libsecret = keyring.backends.libsecret
```